src/clarity-angular/popover/dropdown/dropdown-menu.ts

```diff
@@ -17,7 +18,8 @@ import {Point} from "../common/popover";
     }
 })
 export class DropdownMenu extends AbstractPopover {
-    constructor(injector: Injector, @SkipSelf() parentHost: ElementRef, @Optional() @SkipSelf() nested: DropdownMenu) {
+    constructor(injector: Injector, @Inject(POPOVER_HOST_ANCHOR) parentHost: ElementRef,
```

Comment:
Do we want to do like Willy Wonka, make this `@Optional` and throw our own error if it's not found?

Reply:
We could. Do you want me to add that change with this PR itself or create a new one for it? I think I should create a new one because you already reviewed this one and you would have to review it again.